### PR TITLE
added buttons stylesheets

### PR DESCRIPTION
**Pull request by kjais1720**
https://user-images.githubusercontent.com/66024105/151702699-cd27cdaa-3242-445e-940d-01e85b594fc0.mp4

Preview :

### CODE REVIEW

**Review by sakleprajwal — COMMENTED** [review]
Everything looks good. I recommend You to include a toggle button which will be useful while implementing like/wish-list buttons.

**Comment by sadathshariff** [comment]
Looks good

**Comment by tanishq20** [comment]
Good to merge

File: stylesheets/components/buttons.css [hunk 1]
```diff
@@ -0,0 +1,183 @@
+body{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.buttons{
```
Comment (ttahm3d) [inline stylesheets/components/buttons.css:7]: the name could have been different. I assume you are using this on a div. If thats the case give it a generic name as component-wrapper and use it across the project

**Review by ttahm3d — COMMENTED** [review]
looks good. only one change suggested.

**Comment by nishasen** [comment]
It would be better if you add a light color hover in primary and secondary. Its not that visible.

File: stylesheets/components/buttons.css (hunk 1 above)
Comment (kjais1720) [reply stylesheets/components/buttons.css:7]: Ya, that was just to visualize, I didn't need it in my library I only need the buttons classes here, thanks for pointing it out, I will remove it.

**Pull request merged**